src/core/alert/style.ts

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const AlertsContainer = styled.div`
+    width: 500px;
+    height: auto;
```

Comment:
its implicitly  auto as default so you can remove this line

Reply:
done